editor/src/components/canvas/canvas-strategies/flow-reorder-helpers.ts

```diff
@@ -7,26 +7,45 @@ import {
 } from '../../../core/shared/element-template'
 import { rectContainsPoint, CanvasVector, mod } from '../../../core/shared/math-utils'
 import { ElementPath } from '../../../core/shared/project-file-types'
+import { AllElementProps } from '../../editor/store/editor-state'
 import { stylePropPathMappingFn } from '../../inspector/common/property-path-hooks'
 import { DeleteProperties, deleteProperties } from '../commands/delete-properties-command'
 import { SetProperty, setProperty } from '../commands/set-property-command'
 
-export function isValidSibling(siblingMetadata: ElementInstanceMetadata | null): boolean {
-  // TODO filter float and relative (with TLBR)
-  return !MetadataUtils.isPositionAbsolute(siblingMetadata)
+export function isValidFlowReorderTarget(
+  elementMetadata: ElementInstanceMetadata | null,
+  allElementProps: AllElementProps,
+): boolean {
+  if (MetadataUtils.isPositionAbsolute(elementMetadata)) {
+    return false
+  } else if (elementMetadata?.specialSizeMeasurements.float !== 'none') {
+    return false
+  } else if (MetadataUtils.isPositionRelative(elementMetadata) && elementMetadata != null) {
+    const styleProps = allElementProps[EP.toString(elementMetadata.elementPath)]?.style
```

Comment:
allElementProps uses the spy as the data source, which means that it doesn't guarantee the computedStyle won't have a left top right or bottom set. I think we should use the specialSizeMeasurements here too (I don't know if it need a new field or not)

Reply:
I updated `specialSizeMeasurements` to contain a new field for this